rules/network/command_and_control_download_rar_powershell_from_internet.toml

```
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2020/07/02"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/07"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects a Roshal Archive (RAR) or PowerShell script downloaded from the internet by an internal client. Gaining initial
+access to a system and then downloading encoded or encrypted tools to move laterally is a common practice for
+adversaries as a way to protect their more valuable tools and TTPs. This may be atypical behavior for a managed network
+and can be indicative of malware, exfiltration, or command and control.
+"""
+false_positives = [
+    """
+    Downloading RAR or PowerShell files from the Internet may be expected for certain systems. This rule should be
+    tailored to either exclude systems as sources or destinations in which this behavior is expected.
+    """,
+]
+index = ["packetbeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Roshal Archive (RAR) or PowerShell Files Downloaded from the Internet"
+note = "This activity has been observed in FIN7 campaigns."
+references = [
+    "https://www.fireeye.com/blog/threat-research/2017/04/fin7-phishing-lnk.html",
+    "https://www.justice.gov/opa/press-release/file/1084361/download",
+]
+risk_score = 47
+rule_id = "ff013cb4-274d-434a-96bb-fe15ddd3ae92"
+severity = "medium"
+tags = ["Elastic", "Network"]
+type = "query"
+
+query = '''
+event.category:(network OR network_traffic) AND type:http AND network.transport:tcp AND url.path:/.*(rar|ps1)/ AND source.ip:(10.0.0.0\/8 OR 172.16.0.0\/12 OR 192.168.0.0\/16)
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1105"
+name = "Remote File Copy"
```

```suggestion
name = " Ingress Tool Transfer "
```

It looks like the name of this technique may have changed in ATT&CK. I'm not sure how this currently affects the metadata in our other rules.

---

It shouldn't affect the other rules because the old link redirects to the new techniques. Using the new name will cause the unit tests to fail. You'll probably want to move it back to `Remote File Copy` for now to get it passing.  We need to update all these eventually when, but ECS needs to be updated first and the structure from Kibana. Then we can go back and adjust for the appropriate sub-techniques.

---

Sorry for any confusion - the names need to reflect the new names to pass. I already updated all existing rules, which is why those are not impacted. This is the error from unit tests

```
                      : ATT&CK technique mapping error for rule: ff013cb4-274d-434a-96bb-fe15ddd3ae92 - Roshal Archive (RAR) or PowerShell File Downloaded from the Internet ->
E                       expected: Ingress Tool Transfer for T1105
E                       actual: Remote File Copy
```